packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -27,8 +28,10 @@ class BottomNavigationBarItem {
     this.title,
     Widget activeIcon,
     this.backgroundColor,
+    this.showLabel = true,
```

Comment:
Is there ever a scenario where you want some items to have a label and some to not have a label? If not, maybe this should live on the BottomNavigationBar

Reply:
I've added this attribute separately just in case. However, I think you are right so I moved that into `bottom_navigation_bar`.